composer.json

```diff
@@ -24,17 +24,17 @@
   "require": {
     "php": "^5.5 || ^7.0",
     "psr/log": "^1.0",
-    "php-http/guzzle6-adapter": "^1.1"
+    "php-http/guzzle6-adapter": "^1.1",
+    "daniel-zahariev/php-aws-ses": "^0.8.8",
+    "mailgun/mailgun-php": "^2.0"
   },
```

Comment:
The reason these dependencies were in dev dependencies is because they are soft/suggested dependencies. It's worth exploring alternatives to soft dependencies, the omnipay library uses hard dependencies but separates them per package, for instance omnipay/paypal. But this is out of the scope of this PR so lets bring it back in dev dependencies.